apps/studio/data/api-keys/[id]/api-key-id-query.ts

```diff
@@ -0,0 +1,48 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import { get, handleError } from 'data/fetchers'
+import { ResponseError } from 'types'
+import { apiKeysKeys } from './../keys'
+
+export interface APIKeyVariables {
+  projectRef?: string
+  id?: string
+  reveal: boolean
+}
+
+export async function getAPIKeysById(
+  { projectRef, id, reveal }: APIKeyVariables,
+  signal?: AbortSignal
+) {
+  if (typeof projectRef === 'undefined') throw new Error('projectRef is required')
+  if (typeof id === 'undefined') throw new Error('Content ID is required')
+
+  // @ts-ignore Just a sample here, TS lint will validate if the endpoint is valid
```

Comment:
this isnt needed right? i tried removing the ts-ignore and the lint isn't complaining

Reply:
your right!

Reply:
removed